cache/databroker.go

```diff
@@ -15,9 +20,25 @@ type DataBrokerServer struct {
 
 // NewDataBrokerServer creates a new databroker service server.
 func NewDataBrokerServer(grpcServer *grpc.Server, opts config.Options) *DataBrokerServer {
+	var tlsConfig *tls.Config
+	if opts.DataBrokerCertificate != nil {
```

Comment:
We should still be passing along skipTLS and the CA options without client cert/key set